lua/entities/gmod_wire_expression2/core/custom/applyforce.lua

```diff
@@ -0,0 +1,196 @@
+E2Lib.RegisterExtension("applyforce", false, "Allows E2 chips to applyforce", "Allows E2 chips to applyforce")
+
+local sbox_E2_ApplyForce = CreateConVar( "sbox_E2_ApplyForce", "1", FCVAR_ARCHIVE ) -- 1: Everyone, 2: Only Admins, 3: Only Superadmins
+
+function ApplyForceValidAction()
```

Comment:
> The hook library has nothing to do with this.

A lot of Wiremod code lacks hooks. Please start using hooks to allow more control for 3rd party addons! It is my cons, but please do. Thanks.